backend/src/main/java/com/praksa/auction/service/ProductService.java

```diff
@@ -48,20 +48,19 @@ public List<Product> searchProducts(String search, int count) {
         return productRepository.searchProducts(search, PageRequest.of(0, count));
     }
 
-    public List<Product> getAllProducts(int count) {
+    public List<Product> getAllActiveProducts(int count) {
         return productRepository.findProductsByEndingDateAfter(PageRequest.of(0, count));
     }
 
-
     public String checkSpelling(String search) {
         Hunspell speller = hunspellConfiguration.speller();
         List<String> suggestons = speller.suggest(search);
-        for(String suggest : suggestons) {
-            if(suggest.length() < 3) continue;
+        int minWordLength = 3;
```

Comment:
`private static final int MIN_WORD_LENGTH = 3;
`
and put it before  `private final ProductRepository productRepository;`